azavea/raster-foundry
# Add model search and detail views

## Overview

Add skeletons for market models and search views

### Demo

![anim](https://cloud.githubusercontent.com/assets/4392704/20112366/18e44b24-a5ba-11e6-9bd1-038e4ef87add.gif)

## Testing Instructions

* Start the frontend with `npm start` and navigate to localhost:9091
* Open the marketplace from the browse view by clicking on the Model Market link at the top of the page
* Click search terms to remove them from the list
* toggle tags / categories by clicking them
* Click into a model to view its detail view. Directly opening using a url doesn't work, as there isn't a backend to retrieve it directly, and placeholder data is generated in the search view
* Verify that the carousel works

Closes #627, Closes #628
